OloEngine/src/OloEngine/Core/DebugLevers.cpp

```
@@ -28,6 +33,35 @@ namespace OloEngine::Levers
+        // rendered strings, and at six decimals two genuinely different floats
+        // render identically, so the change notification would be silently
+        // skipped for the very changes that are hardest to notice by eye.
+        [[nodiscard]] std::string RenderFloat(f32 value)
+        {
+            return std::format("{}", value);
+        }
+
+        // Outer whitespace only. The other lever kinds get this for free from
+        // the CVars:: text helpers; OLO_LEVER_NUMBER goes through
+        // ParseNumberLever instead, which must stay strict for the environment.
+        [[nodiscard]] std::string_view TrimForParse(std::string_view text)
```

Cppcheck (2.21.0)

[style] The function 'TrimForParse' is never used.

(unusedFunction)